src/components/SortedList.js

```diff
@@ -1,14 +1,60 @@
 import React from 'react';
 import ListItem from './ListItem';
+import { filter as removePunctuation, isOutOfDate } from '../lib/helper';
 
 const SortedList = ({ itemsKeys, data }) => {
+  const compareItems = (howSoonA, howSoonB, nameA, nameB) =>
+    // compareItems() checks which item has the less number of the estimated days.
+    // returning -1 one for the item whose howSoon is less
+    // means that sort() will put it first and therefore comes first on the list
+    howSoonA < howSoonB
+      ? -1
+      : // the number with the greater number of the estimated days comes second
+      howSoonA > howSoonB
+      ? 1
+      : // reaching this line means that both items
+      // have the same number of estimated days until the next purchase.
+      // so now we have to compare by items name
+      // and since names are unique one name must be greater than the other
+      removePunctuation(nameA) > removePunctuation(nameB)
```

Comment:
I like that you've reused your functions from helper.js. Also that you renamed filter as `removePunctuation`  when you imported it - easier to understand what the function is doing!

Reply:
Thank you so much for the review. :))